source/configure/product-configuration-settings.rst

```diff
@@ -0,0 +1,39 @@
+Product configuration settings
+===============================
+
+.. include:: ../_static/badges/allplans-selfhosted.rst
+  :start-after: :nosearch:
+
+Self-hosted admins can access the following configuration setting in **System Console > Products**, or by editing the ``config.json`` file as described in the following table. 
+
+Boards
+------
+
+.. config:setting:: products-enablepubliclysharedboards
+  :displayname: Enable publicly-shared boards (Products - Boards)
+  :systemconsole: Products > Boards
+  :configjson: .ProductSettings.EnablePublicSharedBoards
+  :environment: MM_PRODUCTSETTINGS_ENABLEPUBLICSHAREDBOARDS
+  :description: Enable the ability to share links to boards publicly with other users.
```

Comment:
This description is missing the options available. See https://github.com/mattermost/docs/blob/master/source/configure/web-server-configuration-settings.rst#forward-port-80-to-443 for an example of how to encode these details.